data/xml/2020.paclic.xml

```diff
@@ -293,14 +295,18 @@
       <author><first>Ran</first><last>Tao</last></author>
       <author><first>Gang</first><last>Peng</last></author>
       <pages>286–293</pages>
-      <url hash="afd6ee5a">2020.paclic-1.33</url>
+      <url hash="21e2f21f">2020.paclic-1.33</url>
+      <revision id="1" href="2020.paclic-1.33v1" hash="afd6ee5a"/>
+      <revision id="2" href="2020.paclic-1.33v2" hash="21e2f21f" date="2021-06-03">2020.paclic-1.33 correction</revision>
     </paper>
```

Comment:
Can you make this more informative? e.g., "Replaced incorrect version of paper" or whatever applies